docs/faq.md

```diff
@@ -0,0 +1,157 @@
+# Feldera Frequently Asked Questions
+
+## What is Feldera
+
+Feldera is an early-stage start-up building a new data processing
+engine for *streaming data*.  In the rest of this document we will
+write "F" for Feldera.
+
+Using indistry jargon, the F engine can be described in several ways:
+
+- a continuous query engine
+- a standing query engine
+- performing incremental view maintenance
+- implementing materialized views
+
+## What is streaming data?
+
+Data that is being continuously modified, usually at a high rate
```

Comment:
Data that continuously changes (i.e., insertions, deletions and updates).

Reply:
I'm not sure that I agree with "modified" or "changes" if one is speaking colloquially. I think that's technically correct, because adding a record is a change, but a lot of streaming data is just additions. So I would tend to include some language about continuously *arriving* data rather than just *changing* data.

Reply:
what arrives can be deletions, I do want to make this point.